dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/src/org/apache/cxf/jaxb/JAXBDataBinding.java

```diff
@@ -457,13 +520,28 @@ protected void justCheckForJAXBAnnotations(ServiceInfo serviceInfo) {
         }
     }
     private void checkForJAXBAnnotations(MessagePartInfo mpi, SchemaCollection schemaCollection, String ns) {
+
+        if (LOG.isLoggable(Level.FINE)) { // Liberty change begin
+           LOG.fine("checkForJAXBAnnotations for MessagePart:" + (mpi != null ? mpi.getName() : "null") );
```

Comment:
is there a reason to call getName() and not just print mpi? Name should be included in the logged contents of mpi.

Reply:
yes, want to reduce the amount of trace logged, just wanted to log the messagepart name